x/emissions/module/rewards/rewards_internal_test.go

```diff
@@ -671,28 +673,100 @@ func TestGetAllReputersOutput(t *testing.T) {
 	}
 	var numReputers int64 = 5
 	wantScores := []alloraMath.Dec{
-		alloraMath.MustNewDecFromString("17.536755245164326"),
-		alloraMath.MustNewDecFromString("20.302662649273707"),
-		alloraMath.MustNewDecFromString("24.278413872561256"),
-		alloraMath.MustNewDecFromString("11.365030585937692"),
-		alloraMath.MustNewDecFromString("15.211816727558011"),
+		alloraMath.MustNewDecFromString("17.46894"),
```

Comment:
should probs comment here that these are the analytically-derived optimal scores that maximize the total consensus score, cuz unclear from variable name "wantScores" otherwise what this should be